tldw_Server_API/app/core/Security/request_id_middleware.py

```
@@ -44,22 +65,20 @@ def __init__(self, app, header_name: str = "X-Request-ID"):
+        request.state.session_id = session_id
 
         # Inject into OTEL baggage for downstream spans/logs
         try:
             from tldw_Server_API.app.core.Metrics.traces import get_tracing_manager
             tm = get_tracing_manager()
             tm.set_baggage("request_id", request_id)
-            if session_id:
-                tm.set_baggage("session_id", session_id)
+            tm.set_baggage("session_id", session_id)
         except Exception:
             pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)